README.md

```diff
@@ -129,17 +151,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Notes**: N/A
 
 ### 2 or more Merged PRs
+#### **OperationCode**
+- **Swag**: Stickers, T-Shirts
+- **Requirements**: 
+    - Resolve 3 issues in any one of our repositories and receive a white or black Operation Code t-shirt!
+    - Merge 2 pull requests in any one of our repositories and receive a small pack of Operation Code stickers!
+- **How to sign up**: [Link](https://docs.google.com/forms/d/e/1FAIpQLSdZAC98DQS7DW1NJT0FrHYTI-88ST6_v8gMlILankqJG295DA/viewform) to claim swag.
+- **Notes**: See the [blog post](https://github.com/OperationCode/START_HERE) for more information.
 
-Nothing here yet!
-
-### 4 Merged PRs
-
-#### **Parity** (Limited Edition Parity Gym Bag)
```

Comment:
Any reason why you're deleting them??

Reply:
Sorry delete by mistake,
will fix it now.